twilio/http/http_client.py

```diff
@@ -47,4 +47,7 @@ def request(self, method, url, params=None, data=None, headers=None, auth=None,
             timeout=timeout,
         )
 
+        self.last_request = request
+        self.last_response = response
```

Comment:
Let's set these to `None` in the constructor so developers can always be sure that the attributes exist.